### PR TITLE
DOCS-15427 System Collection Limitations

### PR DESCRIPTION
**Pull request by kennethdyer**
[DOCS-15427](https://jira.mongodb.org/browse/DOCS-15427)

Adds section to C2C Limitations page on [System Collections](https://docs-mongodbcom-staging.corp.mongodb.com/cluster-sync/docsworker-xlarge/DOCS-15427-limitations/reference/limitations/#system-collections)

[Current Build](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=62c86682f7f8bf6d9fd67546)

### CODE REVIEW

File: source/reference/limitations.txt [hunk 1]
```diff
@@ -101,9 +101,35 @@ Multiple Clusters
 - Syncing one source cluster to many destination clusters is not
   supported.
 
+System Collections
+------------------
+
+{+c2c-product-name+} does not replicate :ref:`system collections 
+<metadata-system-collections>` to the destination cluster.
+
+If the user issues a :dbcommand:`dropDatabase` command on the source cluster,
```
Comment (jmd-mongo) [inline source/reference/limitations.txt:110]: [suggestion]

Let's use second person here.

https://www.mongodb.com/docs/meta/style-guide/writing/write-to-the-user/

File: source/reference/limitations.txt [hunk 2]
```diff
@@ -101,9 +101,35 @@ Multiple Clusters
...
+this change is not direclty applied on the destination cluster. Instead,
+{+c2c-product-name+} drops user collections and views in the destination 
+database, but it does not drop system collections on the destination database.
+
+For example, on the destination cluster:
+
+- The drop operation would not affect a user-created  
+  :data:`system.js <<database>.system.js>` collection. 
+
+- If the user enabled profiling, the :data:`system.profile
```
Comment (jmd-mongo) [inline source/reference/limitations.txt:120]: Similar comment as on line 110.

File: source/reference/limitations.txt [hunk 3]
```diff
@@ -101,9 +101,35 @@ Multiple Clusters
...
+
+For example, on the destination cluster:
+
+- The drop operation would not affect a user-created  
+  :data:`system.js <<database>.system.js>` collection. 
+
+- If the user enabled profiling, the :data:`system.profile 
+  <<database>.system.profile>` collection would remain.
+
+- If the user created views on the source cluster and then dropped the database,
```
Comment (jmd-mongo) [inline source/reference/limitations.txt:123]: Similar comment as on line 110.

File: source/reference/limitations.txt [hunk 4]
```diff
@@ -101,9 +101,35 @@ Multiple Clusters
...
   supported.
 
+System Collections
+------------------
+
+{+c2c-product-name+} does not replicate :ref:`system collections 
+<metadata-system-collections>` to the destination cluster.
+
+If the user issues a :dbcommand:`dropDatabase` command on the source cluster,
+this change is not direclty applied on the destination cluster. Instead,
```
Comment (jmd-mongo) [inline source/reference/limitations.txt:111]: [typo]

"directly"

File: source/reference/limitations.txt [hunk 5]
```diff
@@ -101,9 +101,35 @@ Multiple Clusters
...
+{+c2c-product-name+} drops user collections and views in the destination 
+database, but it does not drop system collections on the destination database.
+
+For example, on the destination cluster:
+
+- The drop operation would not affect a user-created
```
Comment (jmd-mongo) [inline source/reference/limitations.txt:117]: [case suggestion]

Consider using the present tense. 

https://www.mongodb.com/docs/meta/style-guide/writing/use-present-tense/

File: source/reference/limitations.txt [hunk 6]
```diff
@@ -101,9 +101,35 @@ Multiple Clusters
...
+For example, on the destination cluster:
+
+- The drop operation would not affect a user-created  
+  :data:`system.js <<database>.system.js>` collection. 
+
+- If the user enabled profiling, the :data:`system.profile 
+  <<database>.system.profile>` collection would remain.
+
+- If the user created views on the source cluster and then dropped the database,
+  replicating the drop would remove the views, but leave an empty
```
Comment (jmd-mongo) [inline source/reference/limitations.txt:124]: [case suggestion]

Same as comment on line 117.

File: source/reference/limitations.txt [hunk 7]
```diff
@@ -101,9 +101,35 @@ Multiple Clusters
...
+  :data:`system.views <<database>.system.views>` collection.
+
+In these cases, the replicating ``dropDatabase`` would remove all user-created
```
Comment (jmd-mongo) [inline source/reference/limitations.txt:127]: [case suggestion]

Same as comment on line 117.

File: source/reference/limitations.txt (hunk 4 above)
Comment (kennethdyer) [reply source/reference/limitations.txt:111]: Fixed

File: source/reference/limitations.txt (hunk 1 above)
Comment (kennethdyer) [reply source/reference/limitations.txt:110]: Fixed

File: source/reference/limitations.txt (hunk 2 above)
Comment (kennethdyer) [reply source/reference/limitations.txt:120]: Fixed

File: source/reference/limitations.txt (hunk 3 above)
Comment (kennethdyer) [reply source/reference/limitations.txt:123]: Fixed

File: source/reference/limitations.txt (hunk 5 above)
Comment (kennethdyer) [reply source/reference/limitations.txt:117]: Fixed

File: source/reference/limitations.txt [hunk 8]
```diff
@@ -101,9 +101,35 @@ Multiple Clusters
...
+{+c2c-product-name+} drops user collections and views in the destination 
+database, but it does not drop system collections on the destination database.
+
+For example, on the destination cluster:
+
+- The drop operation would not affect a user-created  
+  :data:`system.js <<database>.system.js>` collection. 
+
+- If the user enabled profiling, the :data:`system.profile 
+  <<database>.system.profile>` collection would remain.
```
[the remark this answers is not in the document]
Comment (kennethdyer) [reply source/reference/limitations.txt:121]: Fixed

File: source/reference/limitations.txt (hunk 6 above)
Comment (kennethdyer) [reply source/reference/limitations.txt:124]: Fixed

File: source/reference/limitations.txt (hunk 7 above)
Comment (kennethdyer) [reply source/reference/limitations.txt:127]: Fixed

**Comment by kennethdyer** [comment]
@jmd-mongo, Thanks Joe, Fixes applied.  --Kenneth

File: source/reference/limitations.txt [hunk 9]
```diff
@@ -101,9 +101,35 @@ Multiple Clusters
...
 
+System Collections
+------------------
+
+{+c2c-product-name+} does not replicate :ref:`system collections 
+<metadata-system-collections>` to the destination cluster.
+
+If you issue a :dbcommand:`dropDatabase` command on the source cluster,
+this change is not directly applied on the destination cluster. Instead,
+{+c2c-product-name+} drops user collections and views in the destination
```
Comment (jmd-mongo) [inline source/reference/limitations.txt:112]: [nit]

I'd suggest updating this to be "on the destination database..." to be more consistent with the remaining copy.

**Review by jmd-mongo — COMMENTED** [review]
Thanks for the updates, @kennethdyer! This one LGTM % a nit.

Thanks!
Joe

File: source/reference/limitations.txt (hunk 9 above)
Comment (kennethdyer) [reply source/reference/limitations.txt]: Fixed

**Review by WenbinZhu — APPROVED** [review]
LGTM, thanks!

**Pull request merged**